docs/source/post_processing.md

```diff
@@ -69,20 +76,27 @@ At this step all required `ShapePipe` resulting output files are available in `.
    ```bash
    post_proc_sp -p PSF
    ```
-   to automatically perform a number of post-processing steps. Chose the PSF model with the option `-p psfex|mccd`. In detail, these are (and can also be
-   done individually
+   to automatically perform a number of post-processing steps. Chose the PSF model with the option
+   `-p psfex|mccd`. In detail, these are (and can also be done individually
    by hand):
    
    A. Analyse psf validation files
```

Comment:
You want to have a quick look at the HTML (see below) to see if you are happy with the way this looks.

Reply:
Looks fine!